Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM klakegg/hugo:0.68.3-ext
+FROM docker.io/klakegg/hugo:0.68.3-ext
```

Comment:
docker.io is the default when not specified in docker, but podman by default prompts you for verification when not specified.